src/sensor_mag_plugin.cpp

```diff
@@ -87,7 +87,6 @@ Eigen::Vector3d SensorMagPlugin::getMagFromJSBSim() {
     pitch_rad = _sim_ptr->GetPropertyValue(_jsb_mag_pitch);
     heading_rad = wrap_pi(_sim_ptr->GetPropertyValue(_jsb_mag_hdg));
 
-
```

Comment:
Can you squash this into the previous commit?

Reply:
Done!